addon/components/sortable-group.js

```diff
@@ -3,14 +3,24 @@ import Component from '@ember/component';
 import { set, get, computed } from '@ember/object';
 import { run } from '@ember/runloop';
 import layout from '../templates/components/sortable-group';
+import { mutateDOM, readDOM } from 'ember-batcher';
```

Comment:
do we really need `ember-batcher`? (it can cause side-effects if event will-be re-fired)

Reply:
Without `ember-batcher`, several things wouldn't work properly,
For example,
1. `focus` does not work after we "move" an item. My hypothesis is that it might be scheduled in an earlier DOM operation, so `ember-batcher` allows me to batch the `focus` with the correct DOM update(where the items are swapped). So far it works as intended.
2. Rapid re-firing of an `arrow` event actually causes UI render race conditions. Using `ember-batcher` for `moveItem` actually solved this issue. My understanding is that even if we fire off multiple duplicate events (multiple `mutateDOM` calls), all of them will still be batched for the next DOM update.

Can you elaborate on `side-effects of `ember-batcher`. 
If we are worried about `side-effects` for re-firing, we can always look at other alternatives.

Reply:
I was discussing with some folks and they recommended the `ember-batcher` over `scheduling`. Looks like `scheduleOnce(afterRender` might trigger more unnecessary forced layouts?
However, I did notice that `ember-batccher` is only supported for `ember 3.4`^ so to keep support for `ember-2.8` -> `ember-3.4` maybe I'll need to leverage something else.

Reply:
using `scheduleOnce('render')` for now.